code/UserTag/config.tag

```diff
@@ -0,0 +1,37 @@
+# Copyright 2002-2021 Interchange Development Group and others
+# 
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.  See the LICENSE file for details.
+
+UserTag config Order    key global
+UserTag config Routine  <<EOR
+sub {
+    my ($key, $global) = @_;
+    my $val = $global ? undef : $Vend::Cfg;
+
+    for (split (/[.]/, $key)) {
+        $val // do {
+            local $@;
+            $val = eval (sprintf ('$Global::%s', $_)) // '';
+            ::logError(q{Global config key '%s' produced eval error: %s}, $_, $@)
+                if $@;
```

Comment:
String `eval` is way too risky to use here IMO. If someone were able to inject a config key name they could own the whole system by adding more commands.

Luckily it isn't needed. You can replace lines 17-19 with:

```
no strict 'refs';
$val = ${'Global::' . $_};
```

Reply:
Changed, per discussion, to `${"Global::$_"}`